docs/semgrep-code/findings.md

```diff
@@ -56,327 +98,62 @@ A finding can be categorized in two ways:
-- GitHub Semgrep Cloud Platform installed in the repository that is also added as a project in Semgrep Cloud Platform. For more information, see [Getting started with Semgrep in continuous integration (CI)](/semgrep-ci/overview/).
-:::
-
-To enable triage through comments, follow these steps:
-
-1. In Semgrep Cloud Platform, go to your organization's [Settings](https://semgrep.dev/orgs/-/projects/-/repo-to-scan) page.
-2. Enable the **Triage via comment** <i class="fa-solid fa-toggle-large-on"></i> toggle.
-
-To triage a finding in GitHub, follow these steps:
-
-1. Find an open comment created by Semgrep Cloud Platform in GitHub PR:
-    ![Screenshot of Semgrep Cloud Platform comment in GitHub](/img/semgrep-app-comment-github.png)<br />
-    *Figure 7.* Screenshot of Semgrep Cloud Platform comment in GitHub.
-2. In the comment, reply with:
-    <pre><code>
-    /semgrep ignore <span className="placeholder">&lt;reason&gt;</span>
-    </code></pre>
-3. Substitute the colored placeholder <code><span className="placeholder">&lt;reason&gt;</span></code> with any text that can help to understand why the status of a comment is ignored.
-
-:::info
-Ignoring a finding through a comment in GitHub changes the status of the finding to **ignored** in the Semgrep Cloud Platform. See [Findings](/semgrep-code/findings) page documentation for more details. The GitHub conversation itself is not automatically resolved by this process.
-:::
-
-:::tip
-You can also reopen a finding that was previously ignored. To do so, in step 2. of the preceding procedure, use `/semgrep open`. For `/semgrep open` the reason field is optional.
-:::
-
-<!--
-## Creating Jira tickets from findings
-
-Semgrep supports the creation of Jira tickets from a finding. This enables developers and project managers to create relevant issues within their project or bug-tracking environment. This feature is available to Team/Enterprise Tier users.
-
-To **create a ticket**:
-
-1. Set up a Jira integration through the [Notifications](/semgrep-app/notifications) guide.
-2. Click the **three-dot icon** of the entry.
-3. Click **Create issue with Jira**.
--->
-
-## Use Assistant to analyze findings (beta)
-
-If you have [enabled Assistant](/semgrep-code/semgrep-assistant-code/#enabling-semgrep-assistant),
-you can use the **Analyze** button at the top-right to trigger all Assistant
-functions, including autofix, autotriage, and component tagging. After Assistant
-performs these functions, you can see your results using the **Recommendation**
-or **Component** filters.
-
-To analyze your findings with Assistant:
+## Data retention
 
-1. On the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page, select
-   the findings that you want Assistant to analyze.
-2. Click **Analyze**.
-3. In the confirmation window that appears, confirm that you want to
-   analyze your findings with Assistant.
+The retention period for findings are as follows:
```

Comment:
```suggestion
The retention period for findings is **5 years**.
```
Since this is always true we don't need a table. (The table previously included Community tier, which we sunset.)